examples/jsm/webxr/VRButton.js

```diff
@@ -174,8 +174,6 @@ class VRButton {
 
 	}
 
-	static xrSessionIsGranted = false;
-
 	static registerSessionGrantedListener() {
```

Comment:
Shouldn't we also make this function don't use `static`?

Reply:
Static methods is ES6. So that's okay.